### ISSUE TITLE
Generates a warning about using deprecated feature in React.

### ISSUE THREAD

**Issue by abritinthebay** [nkbt/react-copy-to-clipboard#55]
`Warning: CopyToClipboard: React.createClass is deprecated and will be removed in version 16. Use plain JavaScript classes instead. If you're not yet ready to migrate, create-react-class is available on npm as a drop-in replacement.`

I know you're against ES6 classes but there is another solution: use stateless components!

Looks like the culprit is Component.js and there's nothing there that couldn't be done with Stateless components. 

Thoughts? PRs requests?

**Comment by nkbt** [comment]
Yeah, need to switch to `prop-types` and `create-react-class`. PR welcomed!

**Comment by abritinthebay** [comment]
Would you object to me switching it to a stateless component when possible?
